modules/javafx.web/src/test/java/test/javafx/scene/web/MiscellaneousTest.java

```diff
@@ -473,4 +473,8 @@ void waitForCompletion() {
             assertNull(getEngine().executeScript("window.xmlDoc.body"));
         });
     }
+
+    @Test public void loadJrtCssFileSuccessfully() {
+        getEngine().setUserStyleSheetLocation("jrt:/javafx.web/html/imported-styles.css");
```

Comment:
This won't run as is for a couple reasons. First it's on the wrong thread, but more importantly, you will not be able to load anything from the unit test using a `jrt` URL, so I don't think a unit test in the `javafx.web` module itself will work.

You will either need to provide a system test in `tests/system/` that creates a modular jar file (which is probably overkill for this fix), or else a manual test in `tests/manual`. The latter seems sufficient for this bug and will be easier.

Reply:
Thanks for the feedback.

How is `tests/manual` invoked? I don't see how it would make it possible to load the jrt url successful (as you need to have a modularized app for this, or not?). Moreover, the test here was supposed to only check that jrt url's are no longer rejected. Not that the css is successfully applied (since there is no bug in this part of the code, and I couldn't find such a test for the loading of a normal "file://" or "jar://" either).